doc/cabal-package-description-file.rst

```diff
@@ -1333,6 +1333,16 @@ system-dependent values for these fields.
     from the same package can be specified with the shorthand syntax
     ``my-package:{my-library1,my-library2}``.
 
+    .. Note::
+
+       From version 1.8 of the Cabal specification, the
+       :pkg-field:`build-depends` field of an executable, test suite (from
+       version 1.10) or benchmark (from version 1.14) section can specify the
+       (or, if applicable, a) library in the same package by the name of the
+       library (without a version constraint). Cabal then treats the executable,
+       test suite or benchmark as if it were in another package that depended on
```

Comment:
I find this sentence hard to parse. Did you mean that "Starting from Cabal 1.X relevant components do not get implicit access to the library and should declare such dependency explicitly"?

Reply:
I've reworded and dropped the 'the library/a library' distinction.